codalab/worker/main.py

```diff
@@ -108,9 +108,7 @@ def main():
     parser.add_argument(
         '--shared-file-system',
         action='store_true',
-        help='Internal use: Whether the file system containing '
-        'bundle data is shared between the bundle service '
-        'and the worker.',
+        help='Internal use: If specified, the worker expects to find bundles mounted on its filesystem.',
```

Comment:
Accidentally deleted and rewrote this help string, can probably stash this change.

Reply:
Shouldn't be purely just internal use though since people might run `cl-worker` on the NLP mchines...

Reply:
Yep, that's just how it was initially named, will remove that caveat.